cdap-ui/app/cdap/components/PipelineNodeGraphs/PipelineNodeMetricsGraph.js

```diff
@@ -121,16 +154,103 @@ export default class PipelineNodeMetricsGraph extends Component {
     }
     let recordsInData = data.series.find(d => recordsInRegex.test (d.metricName)) || {};
     let recordsOutData = data.series.find(d => recordsOutRegex.test(d.metricName)) || {};
+    let recordsErrorData = data.series.find(d => recordsErrorRegex.test(d.metricName)) || {};
     recordsInData = (recordsInData.data || []).length ? recordsInData.data[0].value : 0;
     recordsOutData = (recordsOutData.data || []).length ? recordsOutData.data[0].value : 0;
+    recordsErrorData = (recordsErrorData.data || []).length ? recordsErrorData.data[0].value : 0;
 
     return {
       recordsInData,
-      recordsOutData
+      recordsOutData,
+      recordsErrorData
+    };
+  }
+
+  getOutputRecordsForCharting() {
+    return {
+      'recordsOut': {
+        data: this.state.recordsOutData,
+        label: T.translate(`${PREFIX}.recordsOutTitle`),
+        color: RECORDS_IN_PATH_COLOR
```

Comment:
Maybe this is reversed intentionally, but should this have been `RECORDS_OUT_PATH_COLOR`? Similarly, `RECORDS_IN_PATH_COLOR` on line 189?